docs/rtk-query/usage/mutations.mdx

```diff
@@ -103,7 +103,7 @@ The "mutation result" is an object containing properties such as the latest `dat
 
 Below are some of the most frequently used properties on the "mutation result" object. Refer to [`useMutation`](../api/created-api/hooks.mdx#usemutation) for an extensive list of all returned properties.
 
-- `data` - The returned result if present.
+- `data` - The returned result if present. Any previous result will be undefined after triggering a mutation request, so consider component level caching if the previous response data is required for a smooth transition to new data.
```

Comment:
```suggestion
- `data` - The data returned from the latest trigger response, if present. If subsequent triggers from the same hook instance are called, this will return undefined until the new data is received. Consider component level caching if the previous response data is required for a smooth transition to new data.
```

I think this wording is slightly more accurate, as it doesn't actually affect a previous result, it just doesn't return the previous result any more. It hopefully also adds a few more subtle clarifications

Reply:
Agreed. Thank you for the suggestion!